core/benchmarks/KryoBenchmark-results.txt

```diff
@@ -0,0 +1,29 @@
+================================================================================================
+Benchmark Kryo Unsafe vs safe Serialization
+================================================================================================
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 on Mac OS X 10.13.6
+Intel(R) Core(TM) i7-6920HQ CPU @ 2.90GHz
+
+Benchmark Kryo Unsafe vs safe Serialization: Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+basicTypes: Int with unsafe:true                27 /   31        117.9           8.5       1.0X
+basicTypes: Long with unsafe:true               34 /   37         93.8          10.7       0.8X
+basicTypes: Float with unsafe:true              29 /   32        107.6           9.3       0.9X
+basicTypes: Double with unsafe:true             31 /   32        101.3           9.9       0.9X
+Array: Int with unsafe:true                      3 /    3       1130.4           0.9       9.6X
```

Comment:
What do you mean by the following? This line is `9.6X` and the original is `0.3X`.
> The benchmark result is also reason compared to the original one.

Do you mean this is consistent with `master` branch result and the old old (in the comment) is outdate?

Reply:
Hi @dongjoon-hyun , thanks for pointing it out.
This is because the length of the arrays is random
https://github.com/apache/spark/blob/669ade3a8eed0016b5ece57d776cea0616417088/core/src/test/scala/org/apache/spark/serializer/KryoBenchmark.scala#L90-L91

> The benchmark result is also reason compared to the original one.

I will fix the randomness.